instrumentation/lettuce/lettuce-5.1/javaagent/src/main/java/io/lumigo/javaagent/instrumentation/lettuce/v5_1/LettuceInstrumentationModule.java

```diff
@@ -1,54 +0,0 @@
-/*
- * Copyright 2024 Lumigo LTD
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package io.lumigo.javaagent.instrumentation.lettuce.v5_1;
-
-import com.google.auto.service.AutoService;
-import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
-import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
-import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.DefaultClientResourcesInstrumentation;
-import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.LettuceAsyncCommandInstrumentation;
-import net.bytebuddy.matcher.ElementMatcher;
-import java.util.Arrays;
-import java.util.List;
-
-import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
-
-@AutoService(InstrumentationModule.class)
-public class LettuceInstrumentationModule extends InstrumentationModule {
```

Comment:
Bring this class back, when we change an otel instrumentation we write our own InstrumentationModule and change the instrumentation name from lettuce to lumigo-lettuce so it will be clear those spans are with our changes.